.github/workflows/ci-pr-only-team-tags.yml

```diff
@@ -2,6 +2,12 @@ name: Tag Team Reviewers
 
 on:
   pull_request:
+    # Only tag PRs on the branches for which we have protection rules / rulesets
+    branches:
+      - master
+      - 'rc--*'
+      - 'hotfix-*'
+      - 'follow-*'
```

Comment:
since this list might change, would it make sense to instead use an `if` and check for `github.ref_protected`? might be clearer too

https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs